patterns/direct_requestor.py

```diff
@@ -0,0 +1,121 @@
+## Goal: A simpe request pulblisher to publish a request with reply_to set to any topic.
+
+import os
+import platform
+import time
+import calendar;
```

Comment:
Similar comment in the blocking requester

Reply:
Done